systemtest/src/main/java/io/strimzi/systemtest/utils/specific/KeycloakUtils.java

```diff
@@ -22,10 +22,13 @@ public class KeycloakUtils {
 
     private static final Logger LOGGER = LogManager.getLogger(KeycloakUtils.class);
 
+    public final static String PATH_TO_PQO_PREPARE_SCRIPT = "../systemtest/src/test/resources/oauth2/prepare_pg_operator.sh";
     public final static String PATH_TO_KEYCLOAK_PREPARE_SCRIPT = "../systemtest/src/test/resources/oauth2/prepare_keycloak_operator.sh";
     public final static String PATH_TO_KEYCLOAK_TEARDOWN_SCRIPT = "../systemtest/src/test/resources/oauth2/teardown_keycloak_operator.sh";
+    public final static String PATH_TO_PQO_TEARDOWN_SCRIPT = "../systemtest/src/test/resources/oauth2/teardown_pg_operator.sh";
```

Comment:
What does `PQO` mean? `PostgreSQL Operator`?

Also, shouldn't be these two scripts part of the Keycloak preparation scripts?

Reply:
Correct about PQO. 
I was thinking about it, and I decided to split it into separate script. This is not related to keycloak, just how we use it. That's why I decided to split it.

Reply:
So, here you have `PQO`, in the script is `PGO`, what's the point of having it different?

Yeah okay, but IMHO it's, how it is implemented now, just for the Keycloak, right?

Reply:
Correct, I haven't noticed that. I will change PQO -> PGO. Thanks! :)